com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md

```diff
@@ -70,12 +70,12 @@ Just like the Raycasts mentioned earlier, the Grid Sensor can extract any kind o
 ### Example of Grid Observations
 A Grid Observation is best described using an example and a side by side comparison with the Raycasts and the Camera.
 
-Lets imagine a scenario where an agent is faced with 2 enemies and there are 2 "equipable" weapons somewhat behind the agent. Lets also keep in mind some important properties of the enemies and weapons that would be useful for the agent to know. For simplicity, lets assume enemies represent their health as a percentage (0-100%). Lets also assume that enemies and weapons are the only 2 kind of objects that the agent would see in the entire game.
+Let's imagine a scenario where an agent is faced with 2 enemies and there are 2 "equipable" weapons somewhat behind the agent. Some important properties of the enemies and weapons would be useful for the agent to know. For simplicity, let's assume enemies represent their health as a percentage (0-100%). Also assume that enemies and weapons are the only two kinds of objects that the agent would see in the entire game.
```

Comment:
```suggestion
Let's imagine a scenario where an agent is faced with 2 enemies and there are 2 "equipable" weapons somewhat behind the agent. It would be helpful for the agent to know the location and properties of both the enemies as well as the equippable items. For simplicity, let's assume enemies represent their health as a percentage (0-100%). Also assume that enemies and weapons are the only two kinds of objects that the agent would see in the entire game.
```